packages/generator/src/edmx-parser/edmx-file-reader.ts

```diff
@@ -29,7 +31,11 @@ function parseMetadata(
     root
   };
 }
+/* eslint-disable valid-jsdoc */
```

Comment:
Shouldnt this be on line 8 (if it is needed at all)

Reply:
The lint error in this case was only for functions with missing docs. Hence only added here. I changed it to 
`// eslint-disable-next-line valid-jsdoc` here